components/core/src/fs.rs

```diff
@@ -4,6 +4,16 @@ use crate::util::posix_perm::{self,
 #[cfg(windows)]
 use crate::util::win_perm::{self,
                             set_permissions};
+
+#[cfg(windows)]
+use std::os::windows::ffi::OsStrExt;
+
+#[cfg(windows)]
+use winapi::um::winbase::MoveFileExW;
+
+#[cfg(windows)]
+use std::iter;
+
```

Comment:
It would be nice to remove all the newlines from between these use statements.

Reply:
Agreed. I took care of this.